absl/time/duration.cc

```diff
@@ -96,11 +96,9 @@ inline bool IsValidDivisor(double d) {
   return d != 0.0;
 }
 
-// Can't use std::round() because it is only available in C++11.
-// Note that we ignore the possibility of floating-point over/underflow.
 template <typename Double>
 inline double Round(Double d) {
-  return d < 0 ? std::ceil(d - 0.5) : std::floor(d + 0.5);
+  return std::round(d);
```

Comment:
I don't think the comment is about using `std::round()` in the implementation of `Round()`. I think `Round()` should be replaced with `std::round()` now that `std::round()` is available.